File: src/main/java/model/equipment/armor/Boots.java
package model.equipment.armor;

import model.support.allEnums.ArmorMaterial;
import model.support.allEnums.ArmorType;
import model.support.allEnums.EquipmentType;

public class Boots extends Armor {
    public Boots(ArmorMaterial material, double weight, int quality) {
        super(EquipmentType.ARMOR, ArmorType.BOOTS, material, weight, quality);
    }
}

File: src/main/java/model/equipment/armor/Boots.kt
package model.equipment.armor

import model.support.enums.ArmorMaterial
import model.support.enums.ArmorType.BOOTS
import model.support.enums.BodyPart
import model.support.enums.BodyPart.FOOTS
import model.support.enums.EquipmentType.ARMOR

class Boots(material: ArmorMaterial, weight: Double, quality: Int) : Armor(ARMOR, BOOTS, material, FOOTS, weight, quality)